docs/faq.md

```diff
@@ -0,0 +1,261 @@
+
+To keep using the APIs, you can either move your app to production or wait until the free request quota is reset at the beginning of each month.  
+
+
+### Why do I get an error code 429 when I call a Self-Service API?
+
+This error indicates you carried out too many requests and went over your limit of free calls for this API. 
+
+If you wish to keep using the APIs, you can either move your app to production and enter your preferred payment method or alternatively wait until the following month to get more free calls.
+
+### Is the data returned in the Self-Service test environment accurate?
+
+The information returned in test environment is from limited data collections. This is done as a security measure to protect our data and our customers. When you move to production, you will get access to complete and live data.
+
+## Flight Inspiration Search
+
+### Why didn't I get any results for Flight Inspiration Search?
+
+This API works with cached data in the test environment and not all airports are cached. You can find a list of airports included in the cached data in our guides section. When combining these searches with the Airport Nearest Relevant API, it is better to search using the city code rather than the airport code.
+
+## Airport Nearest Relevant API
+
+### Why isn't the Airport Nearest Relevant API returning a specific airport near me?
+
+This may be because an airport is near a national boarder. If so, please check the API parameters for location. Also, please keep in mind that our Airport Nearest Relevant API excludes private and military airports.
+
+## Flight Create Orders API
+
+### How are tickets issued for flights booked with Flight Create Orders in Self-Service?
+
+For Self-Service users, ticketing must be done via airline consolidator. Airline consolidators are essentially air ticket wholesalers that have special arrangements with airlines and, among other functions, can serve as host agencies for travel agents without the necessary IATA/ARC certifications necessary to issue tickets.
+
+To access Flight Create Orders in production, you must have a contract signed with a consolidator for ticket issuance. If you need help finding a consolidator, please contact our support team to put in touch with the best consolidator in your region.
+
+### How can I retrieve booking made with Flight Create Orders in Self-Service?
+
+You can consult booking made through Flight Create Orders using the Flight Order Management API. This API works using a unique identifier(the flight offer id) that is returned by the Flight Create Orders API.
+
+### Does Amadeus pay a commission for flights booked with Flight Create Orders in Self-Service?
+
+Generally, Amadeus does not offer booking commissions for new Self-Service users.
```

Comment:
why its for new Self-Service users and not Self-Service users?

Reply:
@gimperial can you please advise?

Reply:
@nicoongit it's the same conditions for all Self-Service users, regardless if they are old or new. For some reason it was written like that in the FAQ, but I think it can be a bit confusing since it's not fully accurate.

Reply:
Thanks, amended!